### ISSUE TITLE
Last page bug due to Pagination

### ISSUE THREAD

**Issue by techno-disaster** [bartekpacia/mentorship-client#107]
**Describe the bug**

Right now if the last page does not have all 20 members (the number of people we request per page). The members get repeated. This is probably due to a small bug in ```hasReachedMax```, to fill the ```20``` limit.

PS: The data is getting called for all the pages (including the last one) as expected, you can see it in the logs when you scroll to the end. Only the members aren't shown on the app.

**To Reproduce**
Go to members page and scroll to the end.

**Screenshots**

![Screenshot_20200622-084911](https://user-images.githubusercontent.com/52817235/85245478-4d4fe680-b465-11ea-8ba4-b5320737d70b.png)

**Comment by Anirudhk07** [comment]
Can I take this ?

**Comment by techno-disaster** [comment]
Hey @Anirudhk07  sure. assigning it to you

**Comment by techno-disaster** [comment]
@Anirudhk07  Any update on the issue?

**Comment by Akanksha1212** [comment]
Making this issue available for others as there's no response from @Anirudhk07.

**Comment by techno-disaster** [comment]
@bharat-1809 can you take this up?

**Comment by bharat-1809** [comment]
@Techno-Disaster Sure, thanks :)

**Comment by techno-disaster** [comment]
Hey @bharat-1809  any updates?

**Comment by bharat-1809** [comment]
@Techno-Disaster I am still working on it. Sorry for the delay.

**Comment by techno-disaster** [comment]
no problem, we just need a update every 3 days  :) feel free to ask of you are stuck somewhere.

**Comment by bharat-1809** [comment]
@Techno-Disaster I think the issue is already solved. The list didn't get repeated. Please have a look.
![20200713_211408](https://user-images.githubusercontent.com/58745044/87325995-ca93e600-c54f-11ea-897d-78514e679fd2.gif)

**Comment by techno-disaster** [comment]
Yes, i looked into this and found out an issue on the backend, lets keep this on hold for now. Will raise an issue on the backend repo
